statsmodels/sandbox/distributions/tests/test_multivariate.py

```diff
@@ -71,9 +70,9 @@ def test_mvn_mvt_3(self):
         probmvt_R = 0.9522146
         quadkwds = {'epsabs': 1e-08}
         probmvt = mvstdtprob(a2, b, corr2, df, quadkwds=quadkwds)
-        assert_almost_equal(probmvt_R, probmvt, 4)
+        assert_allclose(probmvt_R, probmvt, atol=0.0005)
         probmvn = mvstdnormcdf(a2, b, corr2, maxpts=100000, abseps=1e-5)
-        assert_almost_equal(probmvn_R, probmvn, 4)
+        assert_allclose(probmvn_R, probmvn, atol=0.0001)
```

Comment:
can you write this as 1e-4, above as 5e-4?

Reply:
Changed.